recipes/modflow6/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "modflow6" %}
+{% set version = "6.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MODFLOW-USGS/modflow6/releases/download/{{ version }}/mf{{ version }}.zip
+  sha256: 6023d52dbbdfc22ffb3c85d30322cc56cfde0aa2b59822ab9b03a4d5a381b59c
+  patches:
+    # use inherited environment variables
+    - makefile.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
```

don't you need the ``C`` compiler as well?
```suggestion
    - {{ compiler('fortran') }}
    - {{ compiler('c') }}
```